README.md

```diff
@@ -8,9 +8,50 @@
 Adds support for fulltext search engines like Sphinx and Solr to SilverStripe CMS.
 Compatible with PHP 7.2
 
-## Maintainer Contact
-
-* Hamish Friedlander <hamish (at) silverstripe (dot) com>
+## Important note when upgrading to fulltextsearch 3.7
+
+A `canView()` check against an anonymous user (i.e. someone not logged in) and a `ShowInSearch` check is now performed
+by default against all records (DataObjects) before being added to the search index, and also before being shown in
+search results. This may mean that some records that were previously being indexed and shown in search results will
+no longer appear due to these additional checks.
+
+These additional checks have been added with data security in mind, as it's safer to assume that records failing these
+checks should not be indexed in the first place.
+
+These additional checks also mean that draft-only (unpublished) content will no longer be automatically added to the
+search index as it will now fail the `canView()` check for an anonymous user.  Previously draft-only content could be
+manually excluded from being indexed via `SearchIndex::excludeVariantState()`.  Indexed draft content was filtered from
+showing in public search results via the solr query `&fq=%2B(_versionedstage:"Live"` meaning it only requested content
```

Comment:
I don't think many devs would know what to make of those raw Solr query strings - what's the intent in including it here? Just seems like noise that makes it less likely that devs will keep reading the important stuff below this point :)

Reply:
Have removed